pegasus/sites.v3/code.org/public/poetry.haml

```diff
@@ -0,0 +1,158 @@
+      start_course: hoc_s('poetry_page.poetry_course_content.poem_art.start_course'),
+      start_course_url: CDO.studio_url("/s/poem-art-2021/reset"),
+      start_course_aria_label: hoc_s('poetry_page.poetry_course_content.poem_art.start_course_aria_label'),
+      course_details: hoc_s('call_to_action_see_lesson_details'),
+      course_details_url: CDO.studio_url("/s/poem-art-2021/lessons/1"),
+      course_details_aria_label: hoc_s(:aria_label_call_to_action_module_poem_art)
+    },
+    {
+      overline: hoc_s('poetry_page.poetry_course_content.coding_with_poetry.overline'),
+      title: hoc_s('poetry_page.poetry_course_content.coding_with_poetry.title'),
+      image: "/images/csc-module-coding-w-poetry.png",
+      desc: hoc_s('poetry_page.poetry_course_content.coding_with_poetry.desc'),
+      format: hoc_s('poetry_page.poetry_course_content.coding_with_poetry.format'),
+      start_course: hoc_s('poetry_page.poetry_course_content.coding_with_poetry.start_course'),
+      start_course_url: CDO.studio_url("/s/poetry"),
+      start_course_aria_label: hoc_s('poetry_page.poetry_course_content.coding_with_poetry.start_course_aria_label'),
+      course_details: hoc_s('call_to_action_see_module_details'),
+      course_details_url: CDO.studio_url("/s/poetry?viewAs=Instructor"),
+      course_details_aria_label: hoc_s(:aria_label_call_to_action_module_coding_w_poetry)
+    }
+  ]
+  additional_resources = [
+    {
+      title: hoc_s('poetry_page.additional_resources.try_hoc.title'),
+      image: "/shared/images/social-media/hoc2023_social.png",
+      desc: hoc_s('poetry_page.additional_resources.try_hoc.desc'),
+      url: CDO.code_org_url("/hourofcode"),
+      button_label: hoc_s('poetry_page.additional_resources.try_hoc.button_label'),
+    },
+    {
+      title: hoc_s('poetry_page.additional_resources.NAIPI.title'),
+      image: "/images/naipi_card.png",
+      desc: hoc_s('poetry_page.additional_resources.NAIPI.desc'),
+      url: CDO.code_org_url("/naipi"),
+      button_label: hoc_s('poetry_page.additional_resources.NAIPI.button_label'),
+    },
+    {
+      title: hoc_s('poetry_page.additional_resources.curriculum_catalog.title'),
+      image: "/images/course-catalog-illustration.png",
+      desc: hoc_s('poetry_page.additional_resources.curriculum_catalog.desc'),
+      url: CDO.studio_url("/catalog"),
+      button_label: hoc_s('poetry_page.additional_resources.curriculum_catalog.button_label'),
+    }
+  ]
+
+- js_locale = request.locale.to_s.downcase.tr('-', '_')
+%script{src: webpack_asset_path("js/#{js_locale}/common_locale.js")}
+%script{src: webpack_asset_path('js/code.org/public/poetry.js')}
+
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/page/poetry-styles.css", rel: "stylesheet"}
+
+= view :top_skinny_banner, banner_id: "banner-poetry", banner_url: nil, banner_icon: "fa fa-pen-nib", banner_text: hoc_s('poetry_page.skinny_banner'), light_theme: true
```

Comment:
There's no desired URL to send the user to so the `banner_url` field is `nil`. However, all other uses of `top_skinny_banner` have a `banner_url` so the styling of it expects a link, so if the user hovers over it the text becomes underlined. I wasn't sure which option seemed best for addressing this and was within scope:
1) Update the `top_skinny_banner` element to not do the link styling if no url is supplied
2) Override all the link styling to ensure it doesn't look like a link when under the hood it is
3) No change and just let the text be underlined when a user hovers over it